crabml-core/src/backends/cpu/buf/util.rs

```diff
@@ -15,6 +15,15 @@ pub fn nearest_i32(fval: f32) -> i32 {
     (i & 0x007fffff) - 0x00400000
 }
 
+#[inline]
+pub fn nearest_int(fval: f32) -> isize {
```

Comment:
is this function exactly same as the above one? 

IMHO we could use just reuse the above one